test/rubocop-diff-test

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+FINAL_STATUS=0
+
+function assert_equals() {
+  if [ $# -ne 3 ]; then
+    echo "Incorrect arguments in the followingn assert_equals"
+    echo "  assert_equals $*"
+    exit 255
+  fi
+  local message=$1
+  local actual=$2
+  local expected=$3
+
+  if [ "$actual" == "$expected" ]; then
+    echo "${message}...✅"
+  else
+    echo "-----------------------------------"
+    echo "${message}...❌"
+    echo "  Actual:   '$actual'"
+    echo "  Expected: '$expected'"
+    echo "-----------------------------------"
+    FINAL_STATUS=1
+  fi
+}
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m'
+
+current_path=$(pwd)
+path=~/tmp/rubocop-diff
+rm -rf "$path"
```

Comment:
You create a directory in the user's home directory _without asking_ ??? Nasty. Why not `mktemp -d` ?